### DISCUSSION TITLE
Adding tax/moms for shipping costs

### DISCUSSION THREAD

**Discussion by MartinBjerg** [woocommerce/woocommerce#47379] [Checkout Flow]
Its resly cool that tax automatic is calculated in the final price, but shipping tax costs are excluded, in my country (Denmark) we pay taxes of shipping costs, thats why they must be conected to the total sum, and not as they are now, where its only the goods thats calculeted tax for. In Denmark we call "tax" for "moms" Woocommerce dashbord translate beautiful tax to moms, so only problem is that it dont add moms to shipping costs.

**Comment by senadir** [comment]
Shipping does support tax, just make sure you enabled it:

<img width="573" alt="image" src="https://github.com/woocommerce/woocommerce/assets/6165348/cdb72507-0ded-4160-b75c-938bf08d917c">
<img width="431" alt="image" src="https://github.com/woocommerce/woocommerce/assets/6165348/125cef67-1011-4384-a469-11aab9fa3e13">

<img width="369" alt="image" src="https://github.com/woocommerce/woocommerce/assets/6165348/a1a4f0a5-0e5e-4e53-95e0-1829ad2fbe6d">
